docs/self-managed/setup/guides/connect-to-an-oidc-provider.md

```diff
@@ -41,7 +37,7 @@ configuration](#component-specific-configuration) to ensure the components are c
 <Tabs groupId="authPlatform" defaultValue="generic" queryString values={[{label: 'Generic', value: 'generic' },{label: 'Microsoft Entra ID', value: 'microsoftEntraId' }]} >
 <TabItem value="generic">
 
-### Steps
+<h3>Steps</h3>
```

Comment:
This is to avoid the complications across tabs, yes?

Reply:
Yeah, I figured this page was small enough that they weren't needed in the TOC, and it eliminates the weird duplication/linking.